testdata/ca.crt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I would make the makefile generate those rather than check in but I can do that in followup if you prefer

Reply:
OK. I will reintroduce [this script](https://raw.githubusercontent.com/danehans/fortio/cf67cba6ff13de698df8f7a0febb883c7871cf29/tools/cert-gen), minus the client-side creds. Since the certs are for e2e testing only, should I have `Webtest.sh` call the cert-gen script instead of the Makefile? I will add the cert-gen script and supporting files to the project's root dir, unless you have another preference.

Reply:
that may add a dependency that isn't there though (on openssl unless there is a go way to create certs easily)